docs/dev/analysis/cht-waterfall-chart.rst

```diff
@@ -0,0 +1,812 @@
+          <mc:Choice xmlns:cx1="http://schemas.microsoft.com/office/drawing/2015/9/8/chartex" Requires="cx1">
+            <p:graphicFrame>
+              <p:nvGraphicFramePr>
+                <p:cNvPr id="6" name="Chart 5">
+                  <a:extLst>
+                    <a:ext uri="{FF2B5EF4-FFF2-40B4-BE49-F238E27FC236}">
+                      <a16:creationId xmlns:a16="http://schemas.microsoft.com/office/drawing/2014/main" id="{095E6047-871E-4A6A-93C5-2288E00B8DBB}"/>
+                    </a:ext>
+                  </a:extLst>
+                </p:cNvPr>
+                <p:cNvGraphicFramePr/>
+                <p:nvPr/>
+              </p:nvGraphicFramePr>
+              <p:xfrm>
+                <a:off x="2032000" y="719666"/>
+                <a:ext cx="8128000" cy="5418667"/>
+              </p:xfrm>
+              <a:graphic>
+                <a:graphicData uri="http://schemas.microsoft.com/office/drawing/2014/chartex">
+                  <cx:chart xmlns:cx="http://schemas.microsoft.com/office/drawing/2014/chartex" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" r:id="rId2"/>
+                </a:graphicData>
+              </a:graphic>
+            </p:graphicFrame>
+          </mc:Choice>
+          <mc:Fallback>
+            <p:pic>
+              <p:nvPicPr>
+                <p:cNvPr id="6" name="Chart 5">
+                  <a:extLst>
+                    <a:ext uri="{FF2B5EF4-FFF2-40B4-BE49-F238E27FC236}">
+                      <a16:creationId xmlns:a16="http://schemas.microsoft.com/office/drawing/2014/main" id="{095E6047-871E-4A6A-93C5-2288E00B8DBB}"/>
+                    </a:ext>
+                  </a:extLst>
+                </p:cNvPr>
+                <p:cNvPicPr>
+                  <a:picLocks noGrp="1" noRot="1" noChangeAspect="1" noMove="1" noResize="1" noEditPoints="1" noAdjustHandles="1" noChangeArrowheads="1" noChangeShapeType="1"/>
+                </p:cNvPicPr>
+                <p:nvPr/>
+              </p:nvPicPr>
+              <p:blipFill>
+                <a:blip r:embed="rId3"/>
+                <a:stretch>
+                  <a:fillRect/>
+                </a:stretch>
+              </p:blipFill>
+              <p:spPr>
+                <a:xfrm>
+                  <a:off x="2032000" y="719666"/>
+                  <a:ext cx="8128000" cy="5418667"/>
+                </a:xfrm>
+                <a:prstGeom prst="rect">
+                  <a:avLst/>
+                </a:prstGeom>
+              </p:spPr>
+            </p:pic>
+          </mc:Fallback>
+        </mc:AlternateContent>
+      </p:spTree>
+      <p:extLst>
+        <p:ext uri="{BB962C8B-B14F-4D97-AF65-F5344CB8AC3E}">
+          <p14:creationId xmlns:p14="http://schemas.microsoft.com/office/powerpoint/2010/main" val="4164693524"/>
+        </p:ext>
+      </p:extLst>
+    </p:cSld>
+    <p:clrMapOvr>
+      <a:masterClrMapping/>
+    </p:clrMapOvr>
+  </p:sld>
+
+
```

Comment:
Let's have a look at the slide relationships too, in particular `rId2`. I'm not seeing where `chartEx1.xml` is coming from and that's an unfamiliar root name. Maybe from an example slide that has both a traditional chart and a waterfall chart so we can contrast the two.